scripts/globals/mobskills/hurricane_wing_flying.lua

```diff
@@ -0,0 +1,29 @@
+---------------------------------------------
+--  Hurricane Wing (Airborne Version)
+--
+--  Description: Deals hurricane-force wind damage to enemies within a very wide area of effect. Additional effect: Blind
+--  Type: Magical
+--  Utsusemi/Blink absorb: Wipes shadows
+--  Range: 30' radial.
+--  Notes: Used only by Wyrm when in the air. Only use in a dedicated flying skill set.
+---------------------------------------------
+require("scripts/globals/settings")
+require("scripts/globals/status")
+require("scripts/globals/monstertpmoves")
+
+---------------------------------------------
+function onMobSkillCheck(target, mob, skill)
+    return 0
+end
+
+function onMobWeaponSkill(target, mob, skill)
+    local typeEffect = tpz.effect.BLINDNESS
+
+    MobStatusEffectMove(mob, target, typeEffect, 60, 0, 30)
```

Comment:
Since `typeEffect` is only used in in the `MobStatusEffectMove` call, you can use `tpz.effect.BLINDNESS` directly in it without an assignment.